tests/components/vesync/conftest.py

```diff
@@ -102,3 +102,46 @@ def outlet_fixture():
     """Create a mock VeSync outlet fixture."""
     mock_fixture = Mock(VeSyncOutlet)
     return mock_fixture
+
+
+@pytest.fixture(autouse=True)
+def requests_mock_fixture(requests_mock):
+    """Fixture to build the various responses for the Helpers.call_api method."""
```

Comment:
Can you merge this into `mock_devices_response` please?
I don't think it should be an autouse fixture.